### PR TITLE
Add first Create page and Speech Input

### PR DESCRIPTION
**Pull request by me-jterhorst**
Add the right Header Element to different pages, 
Add Create Page Step One
Add Speechinput

### CODE REVIEW

File: client/src/pages/CreateCase.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import "./CreateCase.css";
+import Card from "../components/Card";
+import HeaderGoForward from "../components/Card_components/HeaderGoForward";
+import SpeechInput from "../components/SpeechInput";
+
+export default function CreateCase() {
+  return (
+    <main id="CreateCase" className="card-screen">
+      <Card header={<HeaderGoForward totalItems={8} />}>
```
Comment (patrikglanemann) [inline client/src/pages/CreateCase.js:9]: Good usage of passing a component as the header.

File: client/src/pages/CreateCase.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        <h2 className="margin-b--s">
+          In case <br /> of
+        </h2>
+        <form className="dispFlex margin-b--l">
```
Comment (patrikglanemann) [inline client/src/pages/CreateCase.js:13]: Does the modifier --s and --l stand for small and large? 
Or you could use a more descriptive name here🤔

**Review by patrikglanemann — COMMENTED** [review]
Good progress!
I saw that there is no navigation bar on the log in page. This is not intended, right?

File: client/src/pages/CreateCase.js (hunk 2 above)
Comment (mikemangas) [reply client/src/pages/CreateCase.js:13]: agree on that.

**Comment by mikemangas** [comment]
looks very nice! one general comment about your (open & close) menu button. your menu slide effect looks really amazing... but i think it is not very user friendly to do the open & close menu button like now. i think some people might get annoyed when they opening the menu, and want to close it again, to move the mouse (or their finger) to the other end of the site...

i hope you understand my point.

<3 keep the good work!!

File: client/src/components/SpeechInput.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import "./SpeechInput.css";
+import { ReactComponent as MicrophoneIcon } from "../svg/icon-microphone.svg";
+
+export default function SpeechInput({ identifier, label }) {
+  return (
+    <fieldset className="Speechinput-box">
```
Comment (AlexJanow) [inline client/src/components/SpeechInput.js:6]: Cool! Kannte fieldset noch nicht :)

**Review by AlexJanow — COMMENTED** [review]
Very clean as always 👍

**Pull request merged**